daemon/pool.go

```diff
@@ -22,7 +24,9 @@ var (
 	// DefaultMaxInstancesPerDriver is the maximum number of instances of
 	// the same driver which can be launched following the default
 	// scaling policy (see DefaultScalingPolicy()).
-	DefaultMaxInstancesPerDriver = runtime.NumCPU()
+	//
+	// Can be changed by setting BBLFSHD_MAX_DRIVER_INSTANCES.
+	DefaultMaxInstancesPerDriver = mustEnvInt("BBLFSHD_MAX_DRIVER_INSTANCES", runtime.NumCPU())
```

Comment:
Shall this be documented somehow on CLI level?

Not sure it it's worth to put in README or directly to the documentation site - WDYT?

Reply:
Added "Environment variables" section to the README. Also exposed `BBLFSHD_MIN_DRIVER_INSTANCES`.